src/defaultConfig.js

```diff
@@ -55,59 +56,63 @@ const config = {
   maxTextSize: 50000,
 
   /**
-   *| Parameter | Description |Type | Required | Values|
-   *| --- | --- | --- | --- | --- |
-   *|fontFamily | specifies the font to be used in the rendered diagrams| String | Required | Trebuchet MS, Verdana, Arial, Sans-Serif |
+   * | Parameter | Description |Type | Required | Values|
+   * | --- | --- | --- | --- | --- |
+   * | fontFamily | specifies the font to be used in the rendered diagrams| string | Required | 'Trebuchet MS', 'Verdana', 'Arial', 'Sans-Serif' |
```

Comment:
I changed all the Values to clearly mark every string by adding `'`

Also, in javascript, types are `number` `string` `boolean`, and the boolean number are `true` `false`. All theses are changed.